lib/slack.py

```diff
@@ -32,10 +32,11 @@ def __init__(self):
         # Cache the client in memory
         self._client = None
 
-    def send_message(self, params):
+    def send_message(self, params, type="Message"):
         # Sends message to the user/channel
+        # Type should either be Message or Ephemeral
         client = self.get_client()
-        client.api_call("chat.postMessage", **params)
+        client.api_call("chat.post" + type, **params)
```

Comment:
This is one of those rare cases where it make sense to append an argument to the string via `+ type` rather than `.format(type)` 👍